### ISSUE TITLE
Log changes in 1.7.2

### ISSUE THREAD

**Issue by heneryville** [dominicbarnes/node-minecraft-runner#6]
It appears the Minecraft server 1.7.2 has changed the message on their logs, including no longer providing a server start message. This may mean that parsing the log files for hints about what's happening inside is no longer going to work :(

**Comment by dominicbarnes** [comment]
Yeah, I've largely abandoned this project because of stuff like this. :(

It's impossible for me to keep up with how often things change, not to mention setting up automated tests.
